core/src/main/java/org/pac4j/jax/rs/pac4j/JaxRsProfileManager.java

```diff
@@ -7,27 +7,31 @@
 
 import javax.ws.rs.core.SecurityContext;
 
-import org.pac4j.core.profile.*;
-import org.pac4j.jax.rs.helpers.RequestPac4JSecurityContext;
+import org.pac4j.core.context.WebContext;
+import org.pac4j.core.context.session.SessionStore;
+import org.pac4j.core.profile.Pac4JPrincipal;
+import org.pac4j.core.profile.ProfileHelper;
+import org.pac4j.core.profile.ProfileManager;
+import org.pac4j.core.profile.UserProfile;
 
 /**
  *
  * @author Victor Noel - Linagora
  * @since 1.0.0
  *
  */
-public class JaxRsProfileManager extends ProfileManager<CommonProfile> {
+public class JaxRsProfileManager extends ProfileManager {
 
-    public JaxRsProfileManager(JaxRsContext context) {
-        super(context);
+    public JaxRsProfileManager(WebContext context, SessionStore sessionStore) {
+        super(context, sessionStore);
     }
 
-    @Override
-    public void logout() {
-        super.logout();
-
-        new RequestPac4JSecurityContext((JaxRsContext) this.context).context().ifPresent(c -> c.principal = null);
-    }
+//    @Override
```

Comment:
To be removed, `logout` no longer exists in the `ProfileManager`. It was too confusing.